pts/model/estimator.py

```diff
@@ -31,10 +31,12 @@ class PyTorchEstimator(Estimator):
     @validated()
     def __init__(
         self, trainer: Trainer, lead_time: int = 0, dtype: np.dtype = np.float32
+      ,**kwargs,
     ) -> None:
         super().__init__(lead_time=lead_time)
         self.trainer = trainer
         self.dtype = dtype
+        self.max_idle_transforms = kwargs["max_idle_transforms"] if "max_idle_transforms" in kwargs else None
```

Comment:
This is by no means wrong but it seems to me that newer versions of gluonts seem to handle this using the `env` variable. If so might be better to stick with it. @kashif

Reply:
ok yes if you can peek into the 0.7.0 branch, you can also see i have merged the implementation of `deepAR` and `deepVAR` as they differ in the output side and the vanilla transformer also works for both univariate and multivariate...

Reply:
Thanks a lot for pointing me to the 0.7.0 branch, really good to know you're activelty working on this. Will have a more thorough look. I realize you're now using the pytorch-lightning trainer (was entertaining doing that).